x/ibc/04-channel/keeper/keeper_test.go

```diff
@@ -192,13 +192,18 @@ func TestKeeperTestSuite(t *testing.T) {
 	suite.Run(t, new(KeeperTestSuite))
 }
 
-func (suite *KeeperTestSuite) createClient(clientID string) {
-	suite.app.Commit()
-	commitID := suite.app.LastCommitID()
+func (suite *KeeperTestSuite) commitNBlocks(n int) {
```

Comment:
I am not familiar with the codebase yet, but this commit N blocks looks like it can be a nice thing to have maybe in the SimApp? Or the simapp helpers if we don't want to add other methods than the abci in SimApp.

I dont know if this is used a lot though.

Reply:
It would be helpful for testing purposes, yes